app/src/main/java/com/swiezaki/swiezakimobile/network/NetworkModule.kt

```diff
@@ -0,0 +1,41 @@
+package com.swiezaki.swiezakimobile.network
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.swiezaki.swiezakimobile.BuildConfig
+import dagger.Module
+import dagger.Provides
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@Module
+class NetworkModule {
+
+    @Provides
+    fun gson(): Gson = GsonBuilder()
+            .create()
+
+    @Provides
+    fun httpClient(): OkHttpClient {
+        val loggingInterceptor = HttpLoggingInterceptor()
+        loggingInterceptor.level = if (BuildConfig.DEBUG) HttpLoggingInterceptor.Level.BASIC else HttpLoggingInterceptor.Level.NONE
```

Comment:
In Kotlin there is no ternary operator (condition ? then : else), because ordinary "if" works fine in this role. So in case of returning value there is no need to create blocks of "if" and "else".